react-native-libraries.json

```diff
@@ -15451,5 +15451,20 @@
     "expoGo": true,
     "newArchitecture": true,
     "fireos": true
+  },
+  {
+    "githubUrl": "https://github.com/marcuzgabriel/react-native-reanimated-skeleton/tree/main/packages/npm",
+    "npmPkg": "react-native-reanimated-skeleton",
+    "examples": [
+      "https://github.com/marcuzgabriel/react-native-reanimated-skeleton/tree/main/packages/app",
+      "https://github.com/marcuzgabriel/react-native-reanimated-skeleton/tree/main/packages/expo",
+      "https://github.com/marcuzgabriel/react-native-reanimated-skeleton/tree/main/packages/web"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "windows": true,
```

Comment:
Hey @metehandemir, thanks for adding new entry to the directory! 👍 

Just one thing, adding out-of-tree platforms support requires providing an example for that platform, see:
* https://github.com/react-native-community/directory?tab=readme-ov-file#%EF%B8%8F-out-of-tree-platforms

I was not able to find `react-native-windows` usage in the whole monorepo. If there is an example outside it, please update the `examples` array, if not let's remove `windows` flag for now, until we have a confirmation.